catroid/src/main/java/org/catrobat/catroid/content/actions/AskSpeechAction.kt

```diff
@@ -0,0 +1,107 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2018 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.catrobat.catroid.content.actions
+
+import android.app.Activity
+import android.content.Intent
+import android.speech.RecognizerIntent
+import android.util.Log
+import com.badlogic.gdx.scenes.scene2d.Action
+import org.catrobat.catroid.content.Sprite
+import org.catrobat.catroid.formulaeditor.Formula
+import org.catrobat.catroid.formulaeditor.InterpretationException
+import org.catrobat.catroid.formulaeditor.UserVariable
+import org.catrobat.catroid.stage.StageActivity
+import org.catrobat.catroid.stage.StageActivity.IntentListener
+import java.util.ArrayList
+
+class AskSpeechAction : Action(), IntentListener {
+    var sprite: Sprite? = null
+    var questionFormula: Formula? = null
+    var answerVariable: UserVariable? = null
+    private var questionAsked = false
+    private var answerReceived = false
+
+    private fun askQuestion() {
+        StageActivity.messageHandler?.obtainMessage(StageActivity.REGISTER_INTENT,
+            listOf(this))?.sendToTarget()
```

Comment:
```suggestion
        StageActivity.messageHandler?.obtainMessage(
            StageActivity.REGISTER_INTENT,
            listOf(this)
        )?.sendToTarget()
```
Kotlin convention is to have a separate line for each argument if they don't fit in one

Reply:
And please remove the unused import